metomi/rose/suite_engine_procs/cylc.py

```diff
@@ -208,7 +208,8 @@ def job_logs_archive(self, suite_name, items):
                 pass
 
     def job_logs_pull_remote(
-        self, suite_name, items, prune_remote_mode=False, force_mode=False
+        self, suite_name, items, prune_remote_mode=False, force_mode=False,
+        rsync=False
```

Comment:
I think the default for `rsync` should be `True` if we are only disabling it for this one case?

Reply:
Fixed, will push later.